src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -82,7 +81,9 @@ class ImageBaseViewer extends BaseViewer {
                 this.loaded = true;
                 this.emit(VIEWER_EVENT.load);
             })
-            .catch(this.errorHandler);
+            .catch(() => {
+                // No-op, this prmise should always resolve
```

Comment:
Seems funky to not track an error if it occurs. Can we remove this, and allow for an uncaught error, then (if it's possible for this to error)?